crates/cli/src/subcommands/local.rs

```diff
@@ -0,0 +1,69 @@
+use crate::config::Config;
+use clap::ArgAction::SetTrue;
+use clap::ArgMatches;
+use clap::{Arg, Command};
+use spacetimedb::stdb_path;
+use std::io::Write;
+use std::path::PathBuf;
+
+pub fn cli() -> Command {
+    Command::new("local")
+        .args_conflicts_with_subcommands(true)
+        .subcommand_required(true)
+        .subcommands(get_subcommands())
+        .about("Manage local SpacetimeDB database")
+}
+
+pub async fn exec(config: Config, args: &ArgMatches) -> Result<(), anyhow::Error> {
+    let (cmd, subcommand_args) = args.subcommand().expect("Subcommand required");
+    exec_subcommand(config, cmd, subcommand_args).await
+}
+
+fn get_subcommands() -> Vec<Command> {
+    vec![Command::new("clear")
+        .about("Deletes all data from the local database")
```

Comment:
```suggestion
        .about("Deletes all data from all local databases")
```

Existing language is unclear; does it delete a single module-and-db or all of them?